spec/integration/shared_examples/active_record_functions.rb

```diff
@@ -0,0 +1,41 @@
+RSpec.shared_examples 'Arproxy does not break the original ActiveRecord functionality' do
+  before do
+    # CREATE
+    ActiveRecord::Base.connection.create_table :products, force: true do |t|
+      t.string :name
+      t.integer :price
+    end
+    # INSERT
+    Product.create(name: 'apple', price: 100)
+    Product.create(name: 'banana', price: 200)
+    Product.create(name: 'orange', price: 300)
```

Comment:
IMHO: How about using `create!` to raise errors if we can't insert records?

Reply:
https://github.com/cookpad/arproxy/pull/32/commits/fbf518bde021a60ae89a2378ba9050a8ad41d931